models/addresses/base/addresses_base_dex.sql

```diff
@@ -0,0 +1,30 @@
+{{config(alias = 'dex',
```

Comment:
please add schema config in the model directly

Reply:
i added a separate schema file again here. lmk if any changes needed